tests/Baseline/hilti.optimization.unimplemented_hook/output.diff

```diff
@@ -1,168 +0,0 @@
-### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
```

Comment:
Does this file need to be deleted? The test doesn't generate it anymore if I'm seeing that right (same presumably for the other tests).

Reply:
I am confused 😅. AFAICT the file _is removed_.